Jint/Runtime/TypeConverter.cs

```diff
@@ -246,7 +249,32 @@ public static uint ToUint32(JsValue o)
         /// <returns></returns>
         public static ushort ToUint16(JsValue o)
         {
-            return (ushort)(uint)ToNumber(o);
+            return (ushort) (uint) ToNumber(o);
+        }
+
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static string ToString(long i)
```

Comment:
separate signatures to catch common integers, preventing wrapping to JsValue overload